### DISCUSSION TITLE
the game not runs my wins 7 x32bits

### DISCUSSION THREAD

**Discussion by azulitougu** [ppy/osu#16004] [Q&A] [answered]
i was trying to install but i get the error that the installation is not compatible,is it because i have 32 bits?

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu#running-osu:

<img width="869" alt="CleanShot 2021-12-08 at 11 15 08@2x" src="https://user-images.githubusercontent.com/22781491/145172750-f9b5a3fe-0549-4576-b8e0-f8402f47e297.png">

osu!(lazer) is no longer supported on Windows 7, please consider updating. Also there is no support for 32-bit architectures yet, see https://github.com/ppy/osu/issues/3984.
